src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -376,6 +376,7 @@
 
 (defn overview
   [id]
+  (rf/dispatch [:communities/fetch-community id])
```

Comment:
leftover?

Reply:
Hi @Parveshdhull, I've addressed above feedbacks. Events in universal links moved to communities.events, infinite loop is fixed